tests/end2end/test_adapter.py

```diff
@@ -467,3 +467,79 @@ def test_adapter_list_assigned_part(dpm_mode_cpcs):  # noqa: F811
             # Cleanup
             if tmp_part:
                 tmp_part.delete()
+
+
+def base_adapter_id(adapter_id, family):
+    """
+    Return the base adapter_id of an adapter_id. Both are in hex.
+    """
+    # Adapter families that have no physical adapter card
+    logical_families = [
+        'hipersocket',
+    ]
+    pchid = int(adapter_id, 16)
+    if family in logical_families:
```

Comment:
your if assigns the same value for then/else ?

Reply:
The code says:
```
    if family in logical_families:
        base_pchid = pchid
    else:
        base_pchid = pchid // 4 * 4
```
That is not the same value.